009_team_7_agent_design/009_team_7_agent_design.tex

```diff
@@ -1 +1,459 @@
+
+\newpage
+The following figures explore scenarios where the agents greediness and kindness are fixed before the food taking routine. 
+
+\subsubsection{Greediness}
+\label{subsubsec: Greediness}
+
+\Cref{fig: Fixed High Greediness} and \Cref{fig: Fixed Low Greediness} illustrate the results from simulations with agents with high greediness ($>$70) and low greediness ($<$30) respectively. All other traits are left as default.
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, fixed high greediness, T7Only, 2000days, 20food, 3deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with high greediness (3 deaths).}
+    \label{fig: Fixed High Greediness}
+\end{figure}
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, fixed low greediness, T7Only, 2000days, 20food, 74deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with low greediness (74 deaths).}
+    \label{fig: Fixed Low Greediness}
+\end{figure}
+
+A tower housing agents with high greediness and enforceable (unbreakable) treaties capabilities show an extremely low death rate suggesting that such a group of agents are able to organise themselves quickly and stabilise the system. A low greediness score sees a dramatic increase in overall deaths and does not achieve stability over the 2000 day simulation.
+
+\subsubsection{Kindness}
+\label{subsubsec: Kindness}
+\Cref{fig: Fixed High Kindness} and \Cref{fig: Fixed Low Kindness} illustrate the results from simulations with agents with high kindness ($>$70) and low kindness ($<$30) respectively. All other traits are left as default.
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, fixed high kindness, T7Only, 2000days, 20food, 153deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with fixed high kindness (153 deaths).}
+    \label{fig: Fixed High Kindness}
+\end{figure}
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[scale=0.25]{009_team_7_agent_design/Images/Cumulative Deaths, fixed low kindness, T7Only, 2000days, 20food, 70deaths.png}
+    \end{center}
+    \caption{Cumulative deaths with fixed low kindness (70 deaths).}
+    \label{fig: Fixed Low Kindness}
+\end{figure}
+
+A high kindness score sees double the deaths of a low kindness score. A high kindness score does not achieve stability however a low kindness score allows for the tower to reach in less than 200. The most likely explanation for this is that agents with a high kindness are potentially too selfless. It is likely that they consume in small quantities when food is available and thus do not have sufficient health when food isn't available. Thus, making them more vulnerable.
```

Comment:
```suggestion
A high kindness score sees double the deaths of a low kindness score. A high kindness score does not achieve stability however a low kindness score allows for the tower to reach it in less than 200 days. The most likely explanation for this is that agents with high kindness are potentially too selfless. It is likely that they consume in small quantities when food is available and thus do not have sufficient health when food isn't available. Thus, making them more vulnerable.
```

Reply:
Not sure if this is the right correction but "reach in less than 200 needs more explanation imo"